src/plugins/intel_gpu/src/plugin/ops/matrix_nms.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "intel_gpu/primitives/matrix_nms.hpp"
+
+#include <memory>
+
+#include "intel_gpu/plugin/common_utils.hpp"
+#include "intel_gpu/plugin/program.hpp"
+
+// clang-format off
```

Comment:
Maybe add a comment about why this clang-format directive is important?

Reply:
It is not necessary, removed

Reply:
done